apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -40,6 +42,12 @@ class LoginTypePicker extends Component {
     });
   };
 
+  recordSectionSetupExitEvent = eventName => {
```

Comment:
I decided to keep this pattern because we use the same function name on other dialog pages, and it avoids requiring a refactor of the rest of the event logging on this page.